pkg/doc/did/doc.go

```diff
@@ -908,12 +915,12 @@ func (doc *Doc) VerifyProof(suites []verifier.SignatureSuite, jsonldOpts ...json
 	return v.Verify(docBytes, append(defaultDocumentLoaderOpt, jsonldOpts...)...)
 }
 
-// VerificationMethods returns verification methods of DID Doc of certain relationship.
+// Verifications returns verification methods of DID Doc of certain relationship.
```

Comment:
Verifications is a misleading name for this accessor.

Reply:
done